js/script.js

```diff
@@ -0,0 +1,253 @@
+//wait until the page is ready to load
+$( document ).ready(function() {
+
+    //populate the day option drop-down with the days of the month up to 31
+    //for each possible day of the month (31)
+    for (var i=0; i<31; i++) {
+        //make an option variable
+        var dayOption = $("<option>");
+        //the option variable's text is the current number (starting at 0) + 1 (to have that number start at 1)
+        dayOption.text(i+1);
+        //if the current number is less than 1
+        if (i<9) {
+            //the current value is equal to the current number (starting at 0) + 1
+            //this way all values will have 2 digits when we use them in our api get
+            dayOption.val("0"+(i+1));
+        }
+        //otherwise there is no 0 in front 
+        else {
+            dayOption.val(i+1);
+        }
+        //add each day option to our input day select
+        $("#input-day").append(dayOption);
+    }
+
+    //get the last year to use as parameters for our API get
+    //make a variable for this year using moment.js
+    var currentYear = Number(moment().format("YYYY"));
+    //subtract 1 from the current year to get last year
+    lastYear = (currentYear-1);
+
+    //object constructor holding the possible inner text for the card as well as the value of the selection
+    function cardText (type, lines, paragraph) {
+        this.type = type;
+        this.lines = lines;
+        this.paragraph = paragraph;
+    }
+    var cardTextOption = [
+        new cardText ("book", ["Hope your birthday is one for the books!","Here’s to your next chapter!","Have a happily-ever-after birthday!","Hope it’s lit(erary)!"], 
+            "This book was one of the top of the best-sellers on your birthday last year!", ),
+        new cardText ("movie", ["You're a VIP!", "Enjoy your day in the spotlight!", "Lights, camera, happy birthday!"], 
+            "Here is a movie from the year you were born that you may not have seen!", ),
+        new cardText ("space", ["You’re out of this world!", "Houston, we have a birthday!", "You've made another voyage around the sun!", "Have a *stellar* birthday!"], 
+            "This was nasa's image-of-the-day on your birthday last year!"),
+        // new cardText ("paper", ["Breaking news, it’s your birthday!", "Experts predict birthday wish comes true!", "I’m very font of you!", "Another birthday? Write on."], "Here is some noteworthy news from the day your were born!"),
+    ];
+
+    //when the user submits their information...
+    $("#submitBtn").on("click", function (){
+        
+        $(".card-title-area").empty();
+        //get the users input name
+        var inputName = $("#input-name").val();
+        //if the user doesn't enter a name, return
+        if (inputName === "") {
```

Comment:
FYI, you can probably shorten this to a falsey check 

```if (!inputName) {```

it's a little more concise and it also deals with false or null